src/restic_compose_backup/backup_runner.py

```diff
@@ -21,11 +20,9 @@ def run(
         image,
         command,
         labels=labels,
-        # auto_remove=True,  # We remove the container further down
         detach=True,
         environment=environment + ["BACKUP_PROCESS_CONTAINER=true"],
         volumes=volumes,
-        network_mode=f"container:{source_container_id}",  # Reuse original container's network stack.
```

Comment:
This line is important because the backup runner must be on the same network as the database containers during a backup. By removing this line, the runner will not have a way to reach the database containers to do the dump if they are not on the default network.

The source container needs to be on the database network and then the runner will reuse that network stack to be on the same network.